scripts/split-e2e-tests-v2.ts

```diff
@@ -108,6 +108,7 @@ const TEST_EXCLUSIONS: { l: string[]; w: string[] } = {
     'src/__tests__/api_3.test.ts',
     'src/__tests__/api_5.test.ts',
     'src/__tests__/custom_policies_container.test.ts',
+    'src/__tests__/custom-resource-with-storage.test.ts',
```

Comment:
We don't run verdaccio on windows. So this test has to be disabled on windows until we ship new extesibility helper package to jump over this break.

See https://github.com/aws-amplify/amplify-cli/issues/12224
![image](https://user-images.githubusercontent.com/5849952/225408903-8b0b9e50-bd91-4bcf-98c4-749f92d9306b.png)